routing/payment_session.go

```diff
@@ -363,3 +366,65 @@ func (p *paymentSession) RequestRoute(maxAmt, feeLimit lnwire.MilliSatoshi,
 		return route, err
 	}
 }
+
+// UpdateAdditionalEdge updates the channel edge policy for a private edge. It
+// validates the message signature and checks it's up to date, then applies the
+// updates to the supplied policy. It returns a boolean to indicate whether
+// there's an error when applying the updates.
```

Comment:
why not just return the error?

Reply:
Yeah I thought about that too. Then I thought it might be better to have the same structure as the method `p.router.applyChannelUpdate`. Plus the error found in this one won't be handled except for logging.